### PR TITLE
Fix: Asc 20390 react native UI kit not getting group details

### PR DESCRIPTION
**Pull request by NaingAmity**

### CODE REVIEW

File: src/screens/Explore/index.tsx [hunk 1]
```diff
@@ -66,7 +66,7 @@ export default function Explore() {
     }, 100);
   };
   const categoryListComponent = () => {
-    const categoryElements = [];
+    const categoryElements: JSX.Element[] = [];
```
Comment (bonanaaaaaa) [inline src/screens/Explore/index.tsx:69]: can we refactor this because imo, we should not use array of elements

Comment (NaingAmity) [reply src/screens/Explore/index.tsx]: refactored Done !

File: src/screens/Explore/index.tsx [hunk 2]
```diff
@@ -60,70 +60,45 @@ export default function Explore() {
...
+  const handleCategoryClick = useCallback(
+    (categoryId: string, categoryName: string) => {
+      setTimeout(() => {
+        navigation.navigate('CommunityList', { categoryId, categoryName });
+      }, 100);
+    },
+    [navigation]
+  );
+
+  const categoryListComponent = useCallback(() => {
```
Comment (bonanaaaaaa) [inline src/screens/Explore/index.tsx:72]:
```suggestion
  const renderCategoryList = useCallback(() => {
```

Comment (NaingAmity) [reply src/screens/Explore/index.tsx]: changed Done krub

**Pull request merged**